src/executor/evaluate/mod.rs

```diff
@@ -320,6 +320,22 @@ async fn evaluate_function<'a, T: 'static + Debug>(
             Nullable::Null => Ok(Value::Null),
         }
         .map(Evaluated::from),
+        Function::Sin(expr) | Function::Cos(expr) | Function::Tan(expr) => {
+            let float_number = eval_to_float(func.name(), expr).await?;
+
+            let trigonometric = |func, value| match func {
+                Function::Sin(_) => f64::sin(value),
+                Function::Cos(_) => f64::cos(value),
+                Function::Tan(_) => f64::tan(value),
+                _ => panic!("Unexpected function: {:?}", func.name()),
+            };
```

Comment:
```rust
let trigonometric = |func, value| match func {
    Function::Sin(_) => f64::sin(value),
    Function::Cos(_) => f64::cos(value),
    _ => f64::tan(value),
};
```
the last branch `_ => panic!(...)` is not reachable.
For now, let's simplify by replacing `Function::Tan(_)` to `_`.